docs/data-access/icgc-25k-data.md

```diff
@@ -92,3 +92,25 @@ Please see ICGC ARGO's [publication policy](https://www.icgc-argo.org/page/77/e3
 ## Questions and Concerns
 
 If you have any further questions or require additional information please contact the [helpdesk](https://platform.icgc-argo.org/contact).
+
+## Frequently asked questions
+
+### 1. I cannot find anything inside the **Example folder**
```

Comment:
This also addresses the syntax for a specific SFTP tool. Can we update this text to include a reference to the tool who's interface is being clarified?

"I am using the `sftp` CLI and cannot find anything inside a directory" or something

Reply:
Updated to [Connecting through CLI SFTP gives the following error no matching host key type found. Their offer: ssh-rsa](https://github.com/icgc-argo/argo-docs/blob/feat/faq/docs/data-access/icgc-25k-data.md#2-connecting-through-cli-sftp-gives-the-following-error-no-matching-host-key-type-found-their-offer-ssh-rsa)